nautobot/ipam/urls.py

```diff
@@ -187,6 +187,12 @@
         name="prefix_changelog",
         kwargs={"model": Prefix},
     ),
+    path(
+        "prefixes/<uuid:pk>/dynamic-groups/",
+        ObjectDynamicGroupsView.as_view(),
+        name="prefix_dynamicgroups",
+        kwargs={"model": Prefix},
+    ),
```

Comment:
Let's make sure we remember to factor this into viewset pattern to be automagic when an object has `dynamic_groups` in its `extras_features`.